components/script/dom/htmlmediaelement.rs

```diff
@@ -1445,7 +1445,12 @@ impl HTMLMediaElement {
             audio_renderer,
             Box::new(window.get_player_context()),
         );
-        let player_id = player.lock().unwrap().get_id();
+
+        let player_mut = player.lock().unwrap();
```

Comment:
Wrap the usage in {} so that the lock guard is dropped as soon as possible. Drop impl will always run a the end of scope.